config/media.php

```diff
@@ -48,4 +48,16 @@
         'width' => 250,
         'height' => 250,
     ],
+
+    /*
+  |--------------------------------------------------------------------------
+  | Preview driver
+  |--------------------------------------------------------------------------
+  |
+  | Make preview (file,blurhash)
+  */
+
+    'drivers' => [
+        'file',
```

Comment:
Can we make it by Enums?
```suggestion
        PreviewType::file,
        PreviewType::blurhash,
```